### ISSUE TITLE
Incorrect Schema for Service_Forwarding

### ISSUE THREAD

**Issue by Random6554** [F5Networks/f5-appsvcs-extension#297]
### Environment
 * Application Services Version: 3.20.0
 * BIG-IP Version: 14.1.2.3

### Summary
Declaration fails when using policyNat as show in the documentation as the schema references it to be excluded.

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/schema-reference.html#service-forwarding-policynat

https://github.com/F5Networks/f5-appsvcs-extension/blob/916a5cf62c5ff0896edfad281c50044cc67b575b/schema/3.20.0/as3-schema-3.20.0-3.json#L27006

Service_Forwarding can be compared to Service_L4 which also has policyNat - https://github.com/F5Networks/f5-appsvcs-extension/blob/916a5cf62c5ff0896edfad281c50044cc67b575b/schema/3.20.0/as3-schema-3.20.0-3.json#L14304

### Steps To Reproduce
1. Submit the following declaration:
```json
    "TEST_Service_Forwarding_123": {
        "class": "Tenant",
        "Application": {
            "class": "Application",
            "template": "generic",
            "test.item-foo.123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234": {
                "class": "Service_Forwarding",
                "policyNAT": {
                    "use": "natPolicy2"
                },
                "virtualPort": 8080,
                "virtualAddresses": [
                    "192.0.2.100"
                ],
                "forwardingType": "l2",
                "layer4": "tcp",
                "profileL4": {
                    "bigip": "/Common/apm-forwarding-fastL4"
                },
                "translateServerAddress": false,
                "translateServerPort": true
            },
            "natPolicy2": {
                "class": "NAT_Policy"
            }
        }
    },
```

2. Observe the following error response:
```json
Error: Error while Sending/Posting http request with AS3 json :{"code":422,"errors":["/TEST_Service_Forwarding_123/test.item-foo.123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234: # is NOT valid"],"declarationFullId":"","message":"declaration is invalid"}  <nil>
```

### Expected Behavior
policyNat should be allowed and configured in as3.

### Actual Behavior
policyNat is no declared and as3 fails.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added AUTOTOOL-1699 to our internal product backlog to look into this further.

**Comment by Sakorah** [comment]
Hi! Is there any info when this is available? I would need this feature to work like it should.

**Comment by dstokesf5** [comment]
This was resolved with the 3.31.0 release of AS3.
